tests/functional/v2/testFn/longTerm.js

```diff
@@ -46,7 +46,7 @@ export const showCorrectOfferInfo = async (contentWindow, modalContent, testName
     await contentWindow.waitForSelector(contentWrapper);
     await contentWindow.waitForSelector(`${offerRow}:first-child`);
     const offerFieldHeadline = await contentWindow.$eval(offerRow, element => element.innerText);
-    const newOfferFieldHeadline = offerFieldHeadline.replace('month.', '').replace(/\n/g, '');
+    const newOfferFieldHeadline = offerFieldHeadline.replace('per month', '').replace(/\n/g, '');
```

Comment:
```suggestion
    const newOfferFieldHeadline = offerFieldHeadline.replace('month.', '').replace('per month', '').replace(/\n/g, '');
```
Small update again for all instances of this regex in the file. I forgot that because different accounts use this same test file we have to account for both cases of the content (checkout and non-checkout).